docs/Benodigde-tabellen.md

```diff
@@ -0,0 +1,20 @@
+Noot: De landelijke tabellen zijn gepubliceerd op https://publicaties.rvig.nl/Landelijke_tabellen/Landelijke_tabellen_32_t_m_61_excl_tabel_35
+
+Voor de BRP-Personen-Bevragen moeten minimaal de volgende tabellen beschikbaar worden gemaakt:
+
+Aanduiding naamgebruik (Waardelijst uit het LO GBA Element 61.10 Aanduiding naamgebruik)
+Aanduiding bij huisnummer (Waardelijst uit het LO GBA Element 11.50 Aanduiding bij huisnummer)
+Aanduiding bijzonder Nederlanderschap (Waardelijst uit het LO GBA Element 65.10 Aanduiding bijzonder Nederlanderschap)
+Aanduiding bijzonder Nederlanderschap
+AdellijkeTitelPredicaat ( Landelijke tabel 38)
+FunctieAdres (Waardelijst uit het LO GBA Element 10.10 Functie Adres)
+Gemeenten (Landelijke tabel 33)
+Geslachtsaanduiding (Waardelijst uit het LO GBA 3.14 Element 04.10 Geslachtsaanduiding)
+Indicatie gezag minderjarige (Landelijke tabel 61)
+Land (Landelijke tabel 34)
+Nationaliteit (Landelijke tabel 32)
+OpschortingBijhouding (Waardelijst uit LO GBA 3.14 Element 67.20 Omschrijving Reden Opschorting Bijhouding)
+OuderAanduiding (Waardelijst gebaseerd op LO GBA 3.14 categorie 02/52 en 03/53. Mogelijke waarden Ouder1 en Ouder2)
+Reden opname/beëindigen nationaliteit (Landelijke tabel 37)
+SoortVerbintenis (Waardelijst uit LO GBA 3.14 Element 15.10 Soort verbintenis)
+Verblijfstitel ( Landelijke tabbel 56)
```

Comment:
hierachter had ik nog de tabel voorvoegsels (tabel 36) verwacht

Reply:
> De tabel voorvoegsels wordt niet gebruikt in BRP-persoon-bevragen

maar we kunnen deze toch alsnog ter beschikking stellen?

Eigenlijk wordt deze wel gebruikt, want we hebben een veld voorvoegsel en een parameter voorvoegsel